notebooks/Example.ipynb

```diff
@@ -30,6 +175,125 @@
+      "\u001b[0;32m~/projects/glue/glue/core/data.py\u001b[0m in \u001b[0;36m__getitem__\u001b[0;34m(self, key)\u001b[0m\n\u001b[1;32m    569\u001b[0m                 \u001b[0;32mraise\u001b[0m \u001b[0mIncompatibleAttribute\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0m_k\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    570\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 571\u001b[0;31m         \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget_data\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mview\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mview\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    572\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    573\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_ipython_key_completions_\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;32m~/projects/glue/glue/core/data.py\u001b[0m in \u001b[0;36mget_data\u001b[0;34m(self, cid, view)\u001b[0m\n\u001b[1;32m   1359\u001b[0m             \u001b[0mcomp\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_externally_derivable_components\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mcid\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1360\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1361\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0mIncompatibleAttribute\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcid\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1362\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1363\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mview\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mIncompatibleAttribute\u001b[0m: Pixel Axis 1 [y]"
+     ]
+    }
+   ],
+   "source": [
+    "from glue.core.data import Data\n",
+    "import numpy as np\n",
+    "\n",
+    "from glue.core.roi import RectangularROI, PolygonalROI, CircularROI, \\\n",
+    "    PointROI, XRangeROI, YRangeROI, AbstractMplRoi\n",
+    "\n",
+    "from glue.core.subset import RoiSubsetState, RangeSubsetState, OrState, \\\n",
+    "    AndState, XorState, MultiOrState, MultiRangeSubsetState\n",
+    "\n",
+    "# data = Data(flux=np.ones((256, 128, 128)), label='Test Flux2')\n",
```

Comment:
```suggestion
    "data = Data(flux=np.ones((256, 128, 128)), label='Test Flux2')\n",
```

This cell fails with this line commented out, should either uncomment it or drop all this subset testing from the generic example notebook.

Reply:
Whoops! Those changes should definitely not be in there. Thanks for catching that, @rosteen!